api_types.ts

```diff
@@ -898,7 +900,6 @@ export type OptionsReference = string & {
 // TODO(dweitzman): Update the type to allow options functions to return a cache TTL.
 /**
  * A JavaScript function for property options.
- * @hidden
  */
 export type PropertyOptionsMetadataFunction<ResultT extends PackFormulaResult[]> = (
```

Comment:
We can also remove a `@hidden` within `PropertyWithOptions` in `schema.ts` to document the `options` property on an object schema

Reply:
Done